### DISCUSSION TITLE
Nominating @sophiewigmore as PHP Maintainer

### DISCUSSION THREAD

**Discussion by thitch97** [paketo-buildpacks/feedback#133] [PHP Team]
I'd like to nominate @sophiewigmore for the role of PHP Maintainer. She has been instrumental in the direction and implementation of the ongoing [PHP CNB Rewrite](https://github.com/paketo-buildpacks/php/issues/485). She continues to show a genuine interest in the future of the PHP CNB and I have no doubt that she would be a stellar addition to the team!

@paketo-buildpacks/php-maintainers 
@paketo-buildpacks/steering-committee 👍/👎?

**Comment by ryanmoran** [comment]
Congrats on becoming a PHP Maintainer, @sophiewigmore!
